object_manager.go

```diff
@@ -12,6 +12,12 @@ type IBObjectManager interface {
 	CreateDefaultNetviews(globalNetview string, localNetview string) (globalNetviewRef string, localNetviewRef string, err error)
 	CreateNetwork(netview string, cidr string, name string) (*Network, error)
 	CreateNetworkContainer(netview string, cidr string) (*NetworkContainer, error)
+	CreateRecordHostWithoutDNS(recordName string, netview string, cidr string, ipAddr string, macAddress string, vmID string) (*RecordHost, error)
```

Comment:
We are writing a plugin for Tterraform which uses this client and at first release there is no plan to support DNS only IPAM. So I don't want to put effort in DNS related stuffs now. 

Reason to use record:host without DNS:  we can't use FixedAddress record because we don't know mac address upfront which is mandatory field to create that. So we decided to use record:host which allocate ip address without mac address.

Reply:
Thanks for the explanation. A couple of things:

1) I am "pretty" sure that you can create FixedAddress record with an empty (00:00:00:00:00:00) MAC address. Please check. If that's true, would that work for your use case?

2) BTW, nit picking, the object is actually called a "Host Record" (although it is specified in WAPI as "record:host"). So, the method should be called CreateHostRecord...

Reply:
fixed address record  with an empty mac address (00:00:00:00:00:00) becomes a address of type reservation,once it becomes type reservation ,attributes /mac address can't be changed/updated.

Reply:
If that is the case, I stand corrected.